examples/pettingzoo/meltingpot_env.py

```diff
@@ -0,0 +1,149 @@
+""" PettingZoo interface to meltingpot environments"""
+
+import dm_env
+from meltingpot.python import substrate
+from ml_collections import config_dict
+from gym import spaces
+import numpy as np
+from functools import lru_cache
+
+from gym.utils import EzPickle
```

Comment:
Please import modules instead of objects directly, i.e.

```
from gym import utils as gym_utils
```

and later use as `gym_utils.EzPickle`